### PR TITLE
more logs

### PR DESCRIPTION
**Pull request by acosferreira**
# Description

Add more logs

Fixes # (THEEDGE-1439)

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [ ] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I run `go fmt ./...` to check that my code is properly formatted
- [ ] I run `go vet ./...` to check that my code is free of common Go style mistakes

### CODE REVIEW

**Comment by acosferreira** [comment]
Merging due no impact

File: pkg/services/images.go [hunk 1]
```diff
@@ -1208,15 +1208,18 @@ func (s *ImageService) SendImageNotification(i *models.Image) (ImageNotification
 		// TODO: formalize message formats
 		recordKey := "ImageCreationStarts"
 		recordValue, _ := json.Marshal(notify)
-		s.log.WithField("message", recordValue).Debug("Preparing record for producer")
+		s.log.WithField("message", recordKey).Debug("Preparing record for producer")
+		s.log.WithField("message", recordValue).Debug("RecordValue")
```
Comment (thom-at-redhat) [inline pkg/services/images.go:1212]: Can these be combined?

File: pkg/services/images.go [hunk 2]
```diff
@@ -1208,15 +1208,18 @@ func (s *ImageService) SendImageNotification(i *models.Image) (ImageNotification
...
 		perr := p.Produce(&kafka.Message{
 			TopicPartition: kafka.TopicPartition{Topic: &topic, Partition: kafka.PartitionAny},
 			Key:            []byte(recordKey),
 			Value:          []byte(recordValue),
 		}, nil)
+		s.log.WithField("message", perr).Debug("after p.Produce")
 		if perr != nil {
-			s.log.WithField("message", perr).Error("Error on produce")
+			s.log.WithField("message", perr.Error()).Error("Error on produce")
+			s.log.WithField("message", perr.Error()).Debug("Error on produce")
```
Comment (thom-at-redhat) [inline pkg/services/images.go:1222]: Do you want both messages being identical at error and debug level?

**Review by thom-at-redhat — COMMENTED** [review]
Please review

**Pull request merged**